pkg/planner/core/physical_plans.go

```diff
@@ -680,26 +680,26 @@ type PhysicalIndexScan struct {
 	// AccessCondition is used to calculate range.
 	AccessCondition []expression.Expression
 
-	Table      *model.TableInfo
-	Index      *model.IndexInfo
+	Table      *model.TableInfo `plan-cache-clone:"shallow"`
```

Comment:
Should we add some comments on what do they mean and list all possible values?

Reply:
I added more comments on genPlanCloneForPlanCacheCode, PTAL